app/(main)/o/[slug]/settings/general-settings.tsx

```diff
@@ -267,9 +267,12 @@ type Props = {
 };
 
 export default function GeneralSettings({ tenant, canUploadLogo }: Props) {
+  const [mounted, setMounted] = useState(false);
   const [isLoading, setLoading] = useState(false);
   const router = useRouter();
 
+  useEffect(() => setMounted(true));
```

Comment:
Looks useful, maybe make a hook for it when you get a chance.

```javascript
const isMounted = useIsMounted();
```

Also can use https://usehooks.com/
as inspiration for hooks in general.